src/actions/user.js

```diff
@@ -0,0 +1,27 @@
+import { apiFetchUserAuthority } from '../api/user'
+import { GenericServerError } from '../modules/error'
+import { RECEIVED_USER, SET_USER_AUTHORITY } from '../reducers/user'
+import { acSetLoadError } from './loader'
+
+export const acReceivedUser = value => ({
```

Comment:
I think that using the word "Received" is possibly a leftover from when we were just learning how to implement redux. (at least I can remember following a tutorial that used that word). Wondering if we want to move away from it and use SET/GET?